### PR TITLE
GitHub Actions and Poetry

### PR DESCRIPTION
**Pull request by vaaaaanquish**
GitHub Actions and Poetry #39 #36

### CODE REVIEW

**Comment by vaaaaanquish** [comment]
@Hi-king @hirosassa Please review. (CI will fail)

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+repository = "https://github.com/m3dev/thunderbolt"
+documentation = ""
+
+[tool.poetry-dynamic-versioning]
+enable = true
+style = "pep440"
+pattern = "^(?P<base>\\d+\\.\\d+\\.\\d+)"
+
+[tool.poetry.dependencies]
+python = "^3.6"
```
Comment (Hi-king) [inline pyproject.toml:18]: 👍

**Comment by Hi-king** [comment]
@vaaaaanquish LGTM!

**Review by Hi-king — APPROVED** [review]

**Review by hirosassa — APPROVED** [review]
Thanks! LGTM

**Pull request merged**